tests/test_config.py

```diff
@@ -212,6 +212,18 @@ def test_invalid_rule_ignore(self):
                                   '  colons:\n'
                                   '    ignore: yes\n')
 
+    def test_invalid_rule_ignore_from_file(self):
+        with self.assertRaisesRegex(
+            config.YamlLintConfigError,
+            'invalid config: ignore-from-file should contain '
+            r'valid filename\(s\), either as a list or string.',
+        ):
+            config.YamlLintConfig(
+                'rules:\n'
+                '  colons:\n'
+                '    ignore-from-file: /invalid_filename\n'
+            )
```

Comment:
I wouldn't bother to use a regex here, since we don't for [other situations](https://github.com/adrienverge/yamllint/commit/2f8ad7003a8571e0c935bc6ba7fef04ed6f3fde6#diff-da24b355349b53e4c8047034767d008e4c2ce9ea6c8933a3acd50d0828f00528R470-R476).

This should be enough:
```python
        self.assertRaises(YamlLintConfigError,
                          config.YamlLintConfig,
                          'rules:\n'
                          '  colons:\n'
                          '    ignore-from-file: 1337\n')
```
(like for the `ignore` case just above)

Reply:
Fixed that.